### DISCUSSION TITLE
Welcome to @ng-idle Discussions!

### DISCUSSION THREAD

**Discussion by grbsk** [HackedByChinese/ng2-idle#162] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by omostan** [comment]
First of all, I would like to commend your team for the great work on this library.

It will be much appreciated if someone from your team kindly releases the latest v12.0.0 to npm, as @makkaronis4e requested in November 2022. The publshed v11.1.0 is compiled with Angular "View Engine" instead of Ivy. Compatibility for the view engine has since been removed from later Angular versions.

Please update the code and publish the latest version on npm.

Thank you very much!
